packages/server/src/app/middleware/confirm-password.validator.js

```diff
@@ -0,0 +1,8 @@
+const { body } = require('express-validator')
```

Comment:
This middleware function is this required to check the confirmed password while a user is registering a new account